e2e-tests/portal/cypress/e2e/journeys/maker-about-supply-contract/fill-in-about-financial-validation.spec.js

```diff
@@ -28,95 +28,121 @@ context('about-buyer', () => {
+    contractAboutSupplier.nextPage().click();
+    contractAboutBuyer.nextPage().click();
     cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().day(), '25');
-
     contractAboutFinancial.preview().click();
 
     contractAboutPreview.errors().should('contain', 'Supply Contract conversion date is required for non-GBP currencies');
     contractAboutPreview.errors().should('contain', 'Supply Contract conversion date Month is required for non-GBP currencies');
     contractAboutPreview.errors().should('contain', 'Supply Contract conversion date Year is required for non-GBP currencies');
 
-    contractAboutFinancial.visit(deal);
-
+    dashboardDeals.visit();
+    cy.clickDashboardDealLink();
+    contract.aboutSupplierDetailsLink().click();
+    contractAboutSupplier.nextPage().click();
+    contractAboutBuyer.nextPage().click();
     cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().month(), '12');
-
     contractAboutFinancial.preview().click();
 
     contractAboutPreview.errors().should('not.contain', 'Supply Contract conversion date Month is required for non-GBP currencies');
 
-    contractAboutFinancial.visit(deal);
-
+    dashboardDeals.visit();
+    cy.clickDashboardDealLink();
+    contract.aboutSupplierDetailsLink().click();
+    contractAboutSupplier.nextPage().click();
+    contractAboutBuyer.nextPage().click();
     cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().year(), '2019');
-
     contractAboutFinancial.preview().click();
 
     contractAboutPreview.errors().should('not.contain', 'Supply Contract conversion date is required for non-GBP currencies');
     contractAboutPreview.errors().should('not.contain', 'Supply Contract conversion date Year is required for non-GBP currencies');
 
-    contractAboutFinancial.visit(deal);
-
-    cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().year(), '{selectall}{backspace}3019');
-
+    dashboardDeals.visit();
+    cy.clickDashboardDealLink();
+    contract.aboutSupplierDetailsLink().click();
+    contractAboutSupplier.nextPage().click();
+    contractAboutBuyer.nextPage().click();
+    cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().year(), '3019');
     contractAboutFinancial.preview().click();
 
     contractAboutPreview.errors().should('contain', 'Supply Contract conversion date cannot be in the future');
 
     const dateTooFarInThePast = thirtyFiveDaysAgo;
 
-    contractAboutFinancial.visit(deal);
+    dashboardDeals.visit();
+    cy.clickDashboardDealLink();
 
-    cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().day(), `{selectall}{backspace}${dateTooFarInThePast.day}`);
+    contract.aboutSupplierDetailsLink().click();
+    contractAboutSupplier.nextPage().click();
+    contractAboutBuyer.nextPage().click();
 
+    cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().day(), `{selectall}{backspace}${dateTooFarInThePast.day}`);
```

Comment:
I do not think you need the selectall and backspace since keyboardInput does selector.clear()

```suggestion
    cy.keyboardInput(contractAboutFinancial.supplyContractConversionDate().day(), dateTooFarInThePast.day);
```
Can this be tested locally if it works and removed everywhere where {selectall}{backspace} exists

Reply:
Done